rosbot_gazebo/launch/simulation.launch.py

```diff
@@ -24,6 +24,13 @@ def generate_launch_description():
     declare_world_arg = DeclareLaunchArgument(
         "world", default_value=["-r ", world_file], description="SDF world file"
     )
+    
+    use_gpu = LaunchConfiguration("use_gpu")
+    declare_use_gpu_arg = DeclareLaunchArgument(
+        "use_gpu",
+        default_value="False",
```

Comment:
I think that it may be better to set it to `True` by default, at least in gazebo simulation, as CPU isn't currently supported.

Reply:
Changed